Dockerfile

```diff
@@ -289,7 +289,7 @@ RUN --mount=type=tmpfs,target=/android \
     cd /android \
     && curl -Os https://dl.google.com/android/repository/commandlinetools-linux-${ANDROID_CMD_VERSION}_latest.zip \
     && unzip -q commandlinetools-linux-${ANDROID_CMD_VERSION}_latest.zip -d $ANDROID_HOME \
-    && PROXY_HOST_AND_PORT=${https_proxy#*://} \
+    && PROXY_HOST_AND_PORT=${https_proxy#*://} PROXY_HOST_AND_PORT=${PROXY_HOST_AND_PORT%/}\
```

Comment:
Can we please put this into separate lines for readability, and apply the same change to `docker/legacy/Dockerfile`, too?

Reply:
Gladly